src/vstart.sh

```diff
@@ -107,7 +107,7 @@ cephx=1 #turn cephx on by default
 cache=""
 memstore=0
 bluestore=0
-rgw_frontend="civetweb"
+rgw_frontend=${rgw_frontend:-"civetweb"}
```

Comment:
this is the first use of `rgw_frontend`, so i don't think this is any different than `rgw_frontend="civetweb"`